nepal_compliance/overrides/salary_slip.py

```
@@ -0,0 +1,175 @@
+)
+
+class CustomPayrollEntry(PayrollEntry):
+    @frappe.whitelist()
+    def submit_salary_slips(self):
+        self.check_permission("write")
+        salary_slips = self.get_sal_slip_list(ss_status=0)
+        
+        if len(salary_slips) > 30 or frappe.flags.enqueue_payroll_entry:
+            self.db_set("status", "Queued")
+            frappe.enqueue(
+                submit_salary_slips_for_employees,
```

Ruff (0.12.2)

Undefined name `submit_salary_slips_for_employees`

(F821)